warmup.py

```diff
@@ -240,17 +254,36 @@ def accumulate(self, L, points):
         The sample is accumulated into the radial distribution average.
 
         """
-        pass
+        if self.rmax >= L/2:
+            raise ValueError('rmax >= L/2')
+        temp_rdf = np.zeros_like(self._rdf)
+        points = np.atleast_2d(points)
+        for i in range(points.shape[0] - 1):
+            for j in range(i+1, points.shape[0]):
+                dist_vect = points[j] - points[i]
+                dist_vect -= L*np.round(dist_vect/L) 
+                dist_mag = np.linalg.norm(dist_vect)
+                if self.rmin <= dist_mag < self.rmax:
+                    bin_coord = int(self.rmin + np.floor(dist_mag/self.dr))
```

Comment:
This is the right idea, but isn't quite right. You need to shift the distance before you floor into a bin.